README.md

```diff
@@ -1,2 +1,19 @@
+Status: Currently in draft and non-binding. 
+
 # Argo Conformance Program
-A standard to ensure interoperability, protect users, and prevent surprises when using commercial offerings with Argo in the name. Currently in draft and non-binding. 
+The Argo community continues to grow dramatically – adding new users, new contributors, and new capabilities. The Argo Conformance Program creates incentives and requriements designed to help the ecosystem grow in a sustainable and user-friendly way. As a diverse set of tools grows around Argo, users need to know when they can they can trust these tools to work together and in a predictable way.
+
+When users use an Argo certified offering they should be able to trust that offering to work with the open source versions of Argo and that configuration designed for Argo will always work.
+
+Software conformance ensures predictability and helps protect the Argo brand. 
+
+# Goals
+* Ensure a consistent and reliable user experience
+* Protect the Argo brand
+* Enable contributions from commercial partners
+* To grow and strengthen the Argo community
```

Comment:
```suggestion
* Ensure contributions from commercial partners
* To grow and strengthen the Argo community
```

We will likely require active maintainers in order to meet conformance. It's not enough to encourage contributions, they should be required.